src/crab/linear_constraints.hpp

```diff
@@ -149,6 +149,9 @@ class linear_expression_t final {
     }
 
     linear_expression_t operator+(int n) const { return this->operator+(number_t(n)); }
+#ifdef _MSC_VER
+    linear_expression_t operator+(uint64_t n) const { return this->operator+(number_t(n)); }
+#endif
```

Comment:
I had tried that but Ubuntu didn't like it unconditionally present:

In file included from /home/dthaler/projects/dthaler/ebpf-verifier/./src/crab/split_dbm.hpp:30,
                 from /home/dthaler/projects/dthaler/ebpf-verifier/src/crab/array_domain.cpp:15:
/home/dthaler/projects/dthaler/ebpf-verifier/./src/crab/linear_constraints.hpp: In member function ‘crab::linear_expression_t crab::linear_expression_t::operator+(uint64_t) const’:
/home/dthaler/projects/dthaler/ebpf-verifier/./src/crab/linear_constraints.hpp:154:88: error: call of overloaded ‘z_number(uint64_t&)’ is ambiguous
  154 | erator+(uint64_t n) const { return this->operator+(number_t(n)); }
      |                                                              ^

Since linear_constraints.hpp should really be rewritten anyway (for issue #149) I had concluded it wasn't worth it to try to track down how to make it work without any ifdefs.